aziot/src/internal/check/mod.rs

```diff
@@ -0,0 +1,127 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+use std::path::PathBuf;
+
+use serde::Serialize;
+use structopt::StructOpt;
+
+mod additional_info;
+mod checks;
+mod util;
+
+pub(crate) use additional_info::AdditionalInfo;
+pub(crate) use checks::all_checks;
+
+// NOTE: this struct gets `structopt(flatten)`ed as part of the `aziot check` subcommand.
+#[derive(StructOpt)]
+pub struct CheckerCfg {
+    // TODO: add aziot version info to https://github.com/Azure/azure-iotedge
+    // /// Sets the expected version of the iotedged binary. Defaults to the value
+    // /// contained in <http://aka.ms/latest-iotedge-stable>
+    // expected_iotedged_version: String,
+    //
```

Comment:
This should be an empty line instead of `//` so that it doesn't connect to the next field's comment.

Reply:
Alas, `rustfmt` strikes again, and will automatically remove the empty line. Of the various alternatives, I found adding this extra `//` the least offensive.

Reply:
Yeesh.